src/dos/program_mousectl.cpp

```diff
@@ -0,0 +1,593 @@
+/*
```

Comment:
This one might have slipped through the clang reformatting cleanup; still using a lot of space-indenting (here's what I get):

![2022-10-21_14-15_1](https://user-images.githubusercontent.com/1557255/197293015-c904901c-7cf6-4a2d-80de-1b2db01d0e66.png)

Reply:
OK, reformatted.